### ISSUE TITLE
`PositionalArgs` are not visible in the class body

### ISSUE THREAD

**Issue by boris-petrov** [typed-ember/glint#159]
```typescript
import Component, { ArgsFor } from '@glint/environment-ember-loose/ember-component';

export interface IFoo {
  readonly PositionalArgs: [foo: string];
}

interface Foo extends ArgsFor<IFoo> {}

class Foo extends Component<IFoo> {
  public static readonly positionalParams = ['foo'];

  public bar(): void {
    // this complains that such a property doesn't exist. One must add:
    // readonly Args: { readonly foo: string };
    // to `IFoo` in order to fix that
    this.foo;
  }
}
```

**Comment by dfreeman** [comment]
This is mentioned in the README as something we can't support, the same way Glint doesn't tie `Element` on your signature to `tagName` and `this.element` on your component 🙂 

For this you'd need to add `declare foo: string` to your class yourself.

**Comment by boris-petrov** [comment]
Sorry, didn't see that in the README. Thanks!
